react-sidebar/react-sidebar-tests.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import * as ReactDom from 'react-dom';
+import Sidebar from 'react-sidebar';
+import { SidebarStyles } from 'react-sidebar';
+
+const sidebar = <ul><li>Item 1</li><li>Item 2</li></ul>;
+const sidebarStyle = {
```

Comment:
I talked to @zhengbli and it looks like the properties of a CSSStyleDeclaration are not meant to be optional. However, you could use `Partial<CSSStyleDeclaration>`. This would require adding `// TypeScript Version: 2.1` as the last line of the header in `index.d.ts`.